setup.cfg

```diff
@@ -32,3 +29,6 @@ filterwarnings=
     ignore: Using or importing the ABCs from 'collections' instead of from 'collections\.abc' is deprecated.*:DeprecationWarning
     ignore: The 'context' alias has been deprecated. Please use 'context_value' instead\.:DeprecationWarning
     ignore: The 'variables' alias has been deprecated. Please use 'variable_values' instead\.:DeprecationWarning
+
+[coverage:run]
+include = starlette/*, tests/*
```

Comment:
```suggestion
include = starlette/*, tests/*
omit = venv/*
```

Reply:
I don't actually understand why we need `omit = venv/*` since that doesn't fall under the `include` paths?

Reply:
It would be better to use source_pkgs = starlette

Reply:
> I don't actually understand why we need `omit = venv/*` since that doesn't fall under the `include` paths?

I blindlessly looked at what we do in uvicorn and indeed I now dont see why we do that :hamburger: